unified-codes/apps/web/src/components/templates/ErrorPage.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+import { Grid } from '@unified-codes/ui';
+const CatOnBike = require('../../assets/cat-on-bike.png');
+
+export interface ErrorPageProps {
+  code: string;
+}
+
+export type ErrorPage = React.FunctionComponent<ErrorPageProps>;
+
+export const ErrorPage: ErrorPage = ({ code }) => {
+  switch (code) {
+    case '404':
```

Comment:
I think the `switch` statement would be something to lift out to the parent (i.e. don't couple the component with a specific `404` error or whatever, can always make another component for another use-case?